Sources/Hedera/Execute.swift

```diff
@@ -91,23 +91,26 @@ private struct ExecuteContext {
     fileprivate let grpcTimeout: Duration?
 }
 
-internal func executeAny<E: Execute & ValidateChecksums>(_ client: Client, _ executable: E, _ timeout: TimeInterval?)
+internal func executeAny<E: Execute & ValidateChecksums>(
+    _ client: Client, _ executable: E, _ timeout: TimeInterval?, _ initialTransactionId: TransactionId?, _ index: Int
+)
```

Comment:
name the two new args:
```suggestion
internal func executeAny<E: Execute & ValidateChecksums>(
    _ client: Client,
    _ executable: E,
    _ timeout: TimeInterval?,
    initialTransactionId: TransactionId?,
    index: Int
)
```
Although, I highly discourage passing more arguments through this function, all this information should exist on the executable already

Reply:
I've added two fields, `firstTransactionId` and `index` to `Execute` protocol to expose `initialTransactionId` and `currentChunk` and removed the extra arguments.